CHANGELOG.unreleased.md

```diff
@@ -15,6 +15,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [XOSAN] Fix the installer (PR [#4839](https://github.com/vatesfr/xen-orchestra/pull/4839))
+- [OVA Import] Fix the minimum static memory of imported VM (PR [#4866](https://github.com/vatesfr/xen-orchestra/pull/4866))
```

Comment:
IMHO, this is not enough user oriented, what was the problematic behavior?

Reply:
Yeah, that's why I didn't want to put a message.
By default the minimum was 128MB which could cross over if the imported VM asked for less than that. Which is mostly visible with very small test VMs.

Reply:
What do you think about this one?

```suggestion
- [OVA Import] Fix *no host available* error when starting for imported VMs with low memory (PR [#4866](https://github.com/vatesfr/xen-orchestra/pull/4866))
```